storage/api/csk_operations.py

```diff
@@ -2,14 +2,14 @@
 
 # Copyright 2016 Google Inc. All rights reserved.
 #
-# Licensed under the Apache License, Version 2.0 (the "License");
+# Licensed under the Apache License, Version 2.0 (the 'License');
```

Comment:
Lol, please leave the license header unmodified - this will probably throw off our automated license scanners. ;)